Child Minor Addendum.md

```diff
@@ -0,0 +1,34 @@
+**<h1>CHILD MINOR ADDENDUM</h1>**
```

Comment:
Could you rename this file to ChildMinorAddendum.md (without spaces) just to avoid any possible encoding issues?

Reply:
Done